src/BlobField/index.js

```diff
@@ -3,7 +3,7 @@ import { useHistory } from "react-router-dom";
 import config from "src/config";
 import paper from "paper";
 
-import style from "./style.module.scss";
+import './style-blobs.css';
```

Comment:
This managed to fix the styling when the user enters the homepage from and external link or by typing in the address bar.

Reply:
very strange that this would happen here and not break the other components that use css modules.

Reply:
Yeah exactly my thought.

Reply:
I can't reproduce this. I renamed style-blobs.css to style.module.css, removed `blobs-` from the style names. and then 
```
import {
  wrapper,
  popover,
} from './style.module.css';
```
in the index.js and then `className={wrapper}` etc and the correct styles are being applied. It does appear as `style_wrapper__1dMCn` but that's just the generated name from the css modules. The content of the class is accurate.

Reply:
or was this only happening on Production? Just realized that

Reply:
This is happening in production. Also I have temporarily fixed it by removing the class assignment on PageHeader. Have a look at the last commit when you can https://github.com/blerchin/dma-20-mfa-show/commit/7767417d3a2dc8bf9c8fecc5ecf6f050c2cdd40e